### PR TITLE
Add pyaes recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/anyio/meta.yaml [hunk 1]
```diff
@@ -1,41 +0,0 @@
-{% set name = "anyio" %}
-{% set version = "1.0.0" %}
```
Comment (scopatz) [inline recipes/anyio/meta.yaml]: Please restore other recipes or merge from master

**Comment by mwilson8** [comment]
@scopatz recipes restored

File: recipes/pyaes/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  license_file: "LICENSE.txt"
+  summary: "Pure-Python Implementation of the AES block-cipher and common modes of operation"
+  doc_url: "https://pypi.org/project/pyaes/"
+  dev_url: "https://github.com/ricmoo/pyaes"
+
+extra:
+  recipe-maintainers:
+    - mwilson8
+    - oblute
+    - benhuff
```
Comment (scopatz) [inline recipes/pyaes/meta.yaml]: @oblute @benhuff - are you willing to be maintainers?

Comment (benhuff) [reply recipes/pyaes/meta.yaml]: yes!

Comment (oblute) [reply recipes/pyaes/meta.yaml]: yes

**Comment by mwilson8** [comment]
@marcelotrevisani this is ready for final review & merge

**Pull request merged**